implementation/src/main/java/io/smallrye/faulttolerance/HystrixInitializer.java

```diff
@@ -55,6 +58,7 @@ void onStartup() {
         LOGGER.info("Hystrix concurrency strategy used: " + strategy.getClass().getSimpleName());
         HystrixPlugins.getInstance().registerConcurrencyStrategy(strategy);
         HystrixPlugins.getInstance().registerCommandExecutionHook(new FaultToleranceCommandExecutionHook());
+        TracingInstaller.install();
```

Comment:
Why not pass `HystrixPlugins.getInstance()` into the `TracingInstaller` so it doesn't need to reset all the plugins and re add them again?

Reply:
The code was designed to work without passing strategies and plugins around. If we "own" the strategy installation we can go around without resetting. 

Is this code executed always even when a user provides custom concurrency strategy?